### ISSUE TITLE
Borrow a Book system

### ISSUE THREAD

**Issue by RichardLitt** [rektide/archive-eldergods-com#1]
How does it work? Is this just HTML?

**Comment by rektide** [comment]
Yeah I hand authored some HTML. I was advertising the site over Physical Web/Eddystone for a while.

If I got a couple more nibbles, I would have gone and implemented a really simple web form to get us in touch with each other- probably a twitter bot? I did see some traffic, but I only saw one person ever click Borrow in my logs. Another person or two hitting borrow and I think I would have had more motivation to keep cracking on this. I'm 98% certain all hits were via people who'd found the site via Eddystone.

Aside from Eddystone, there is one other flourish- everything is semantic. I used RDFa to markup items-
* the page is a http://schema.org/OfferCatalog
* each book is a http://schema.org/Book
* books have  potentialActions of http://schema.org/BorrowAction

We don't really have savvy User Agents that understand the objects on the page, atm, but ideally I'd like for someone being able to visit this page have the user agent be able to tell them there's a bunch of books they could potentially borrow, without them even looking at the page.

So to sum up- it doesn't really work. It's far from complete- there's no way to talk, to organize a rendezvous where people get to borrow these books. It's also in need of some at least minimal styling and better copy. But like 87 people went to the site, without me telling anyone or it really being visibile on the web, so that was kind of cool, and if they had some super-advanced web browser from 2033 they would maybe have been impressed.

Questions? Thoughts?

**Comment by RichardLitt** [comment]
Cool. That answers my questions - basically, it isn't extensible for my purposes. I'm interested in the idea of sharing books. I often send books to friends, and would like more friends to ask me for books. So, I was curious if there was a nice system you had I could borrow. It doesn't sound like it. That's OK! I can use the written word pretty adequately - I'm not sure that your borrow system improved on an email link (no offense, just trying to be minimal).

Thank you so much for the explanation! I'll close this now.
